dev/ci/dist.sh

```diff
@@ -2,8 +2,7 @@
 
 set -ex
 
-yarn install:electron
-yarn install:elm
+yarn install:all
```

Comment:
I think we should avoid `yarn install:all` here, as it executes again a `yarn install` that is already done before by travis: it slows down the build.

Reply:
It won't take more than a few seconds to detect there aren't any modules to install while it helps in avoiding to forget some install phase (or to remove one of them even though that shouldn't happen often).